sql/testdata/builtin_function

```diff
@@ -84,3 +84,107 @@ SELECT SUBSTR('12345', -2, 77)
 
 statement error SUBSTR: negative substring length -1 not allowed
 SELECT SUBSTR('12345', 2, -1)
+
+query T
+SELECT concat_ws(',', 'abcde', 2, NULL, 22)
+----
+abcde,2,22
+
+query T
+SELECT split_part('abc~@~def~@~ghi', '~@~', 2)
+----
+def
+
+query T
+SELECT repeat('Pg', 4)
+----
+PgPgPgPg
+
+query T
+SELECT repeat('Pg', -1) || 'empty'
+----
+empty
+
+query T
+SELECT ascii('x')
+----
+120
+
+query T
+SELECT md5('abc')
+----
+900150983cd24fb0d6963f7d28e17f72
+
+query T
+SELECT to_hex(2147483647)
+----
+7fffffff
+
+query T
+SELECT strpos('high', 'ig')
+----
+2
+
+query T
+SELECT btrim('xyxtrimyyx', 'xy')
+----
+trim
+
+query T
+SELECT btrim('   postgres  ')
```

Comment:
You can concatenate a string to the front a back to verify the `btrim` result has no whitespace:

```
SELECT 'a' || btrim('    postgres    ') || 'b'
----
apostgresb
```